nav2_controller/src/nav2_controller.cpp

```diff
@@ -65,10 +61,14 @@ ControllerServer::on_configure(const rclcpp_lifecycle::State & state)
 {
   RCLCPP_INFO(get_logger(), "Configuring controller interface");
 
-  get_parameter("controller_plugin_ids", controller_ids_);
-  get_parameter("controller_plugin_types", controller_types_);
-  get_parameter("controller_frequency", controller_frequency_);
+  get_parameter("plugins", controller_ids_);
+  // Default to DWB if no controller plugin is provided
+  if (controller_ids_.empty()) {
```

Comment:
Rather than this, you could just `declare_parameter("plugins", "FollowPath")` and the counterpart for DWB. That will add as the default if not included in the configuration file, and stop you from having to do all this `if empty` logic since it won't be empty ever then.